add geolocation samples (#944)
GoogleChrome/chrome-extensions-samples: functional-samples/cookbook.geolocation-contentscript/README.md

This recipe shows how to get `geolocation` access within a [content script][5]

## Context

> In extensions, geolocation uses the same web APIs you are used to. Instead, if you want to get geolocation information, use the same [`navigator.geolocation`](geolocation) DOM API that any web site normally would. This demo shows how to access this data in a [content script][6]. We also have demos for a [Service Worker][4](via [offscreenDocument][2], and using a [popup][3]

Reviewer: ```suggestion
To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would. This demo shows how to access this data in a [content script][5]. We also have demos for a [Service Worker][4](via [offscreenDocument][2], and using a [popup][3].
```

## Running this extension

1. Clone this repository.

> 1. Load this directory in Chrome as an [unpacked extension][2].

Reviewer: ```suggestion
1. Load this directory in Chrome as an [unpacked extension][1].
```

1. Open the Extension menu and click the extension named "Geolocation - content script".

[1]: https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked
[2]: https://developer.chrome.com/docs/extensions/reference/offscreen/
[3]: functional-samples/cookbook.geolocation-popup
[4]: functional-samples/cookbook.geolocation-offscreen

> [5]: https://developer.chrome.com/docs/extensions/mv3/content_scripts/

Reviewer: ```suggestion
[5]: https://developer.chrome.com/docs/extensions/mv3/content_scripts/
[6]: https://developer.mozilla.org/docs/Web/API/Navigator/geolocation
```

This recipe shows how to get `geolocation` access within a [content script][5]

> ## Context

Author: I add a title and change context to overview - any other places you would like to see more alignment?

To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would. This demo shows how to access this data in a [content script][5]. We also have demos for a [Service Worker][4](via [offscreenDocument][2], and using a [popup][3].

## Running this extension

1. Clone this repository.
1. Load this directory in Chrome as an [unpacked extension][1].
1. Navigate to example.com
1. Inspect the page to see the location being logged to the console.

[1]: https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked
[2]: https://developer.chrome.com/docs/extensions/reference/offscreen/

> [3]: functional-samples/cookbook.geolocation-popup

Reviewer 2: I'm not sure that these relative links will work unless we add a / to make them relative to the root?

Author: fixed!

> This recipe shows how to get `geolocation` access within a [content script][5]

Reviewer 2: Nit: Can we move this down between the header and "Overview"?

Author: fixed with [1dbac70](https://github.com/GoogleChrome/chrome-extensions-samples/pull/944/commits/1dbac7034801f93237ddfa43f3a9a44f9f7552eb)
